internal/events/event.go

```diff
@@ -153,3 +179,20 @@ func toProps(attrs ...interface{}) (result string, err error) {
 
 	return "", err
 }
+
+type BaseEvent interface {
+	GetName() string
+	GetSeverity() string
+	FormatMessage() string
+}
+
+type ClusterEvent interface {
+	BaseEvent
+	GetClusterId() *strfmt.UUID
+}
+
+type HostEvent interface {
```

Comment:
We are adding late binding feature, some host events are related to infra env and not to a specific cluster. 
cc: @yevgeny-shnaidman

Reply:
`GetClusterID` is really a convenience here though, right? Once we have the host we could get the cluster easily enough. Alternatively `GetClusterID` could return `nil` :man_shrugging: Maybe it needs an error return value as well?

Reply:
For late binding we would have another event type, UnboundHostEvent, right?